### DISCUSSION TITLE
Create a new package : PC Manager

### DISCUSSION THREAD

**Discussion by labrute** [bcurran3/ChocolateyPackages_CCR#336] [Ideas]
I am writing to you today with a request regarding the creation of a Chocolatey package for the Microsoft PC Manager application.

As an enthusiast and user of Microsoft products, I have come across the PC Manager application, which offers valuable system management and optimization features.

If you are available and willing to undertake this task, I kindly request your assistance in creating a Chocolatey package for Microsoft PC Manager. Your expertise in package creation and familiarity with the Chocolatey ecosystem would be instrumental in ensuring a seamless installation experience for the community.

Please let me know if you require any specific information or resources from my end to facilitate the creation of this package. I am more than happy to collaborate and provide any necessary details to support the successful development of the Chocolatey package for Microsoft PC Manager.

PC Manager web site : [PC Manager](https://pcmanager-en.microsoft.com/)

Thank you for considering my request. 

Best regards,

**Comment by bcurran3** [comment]
Bonjour JF.

I just read about this last night in my news feed. Usually something like this that gets media attention is packaged up rapidly by someone. I just checked the moderator queue and I don't see it pushed yet. I also verified it's easy to silently install. I'll see about maybe getting it packed up this weekend.

**Reply by labrute** [reply]
Hello,

Thanks, if I can help you in any way let me know.

**Comment by bcurran3** [comment]
Closing this as I am no longer creating or maintaining Chocolatey packages. Sorry, out
of my control.

Hopefully you've enjoyed my work and found it useful.

[![Bella Ciao!](https://img.youtube.com/vi/9ao4FEaDGhQ/0.jpg)](https://www.youtube.com/watch?v=9ao4FEaDGhQ)

I'm sure you know this but Chocolatey has a Google Groups where you can request someone to package up and maintain a program. If this isn't already in the CCR, you should request it there.
